scripts/tests.sh

```diff
@@ -100,29 +101,29 @@ do
         fi
         for abiv1 in $FORCE_ABIV1_RUNS
         do
-            force_abiv1_flag=""
+            force_abiv1_flag=()
             if [[ "$abiv1" == "yes" ]]
             then
-                force_abiv1_flag="--abiencoderv1"
+                force_abiv1_flag=(--abiencoderv1)
             fi
-            printTask "--> Running tests using $optimize --evm-version $vm $force_abiv1_flag..."
+            printTask "--> Running tests using $optimize --evm-version $vm ${force_abiv1_flag[*]}..."
 
-            log=""
+            log=()
             if [ -n "$log_directory" ]
             then
                 if [ -n "$optimize" ]
                 then
-                    log=--logger=JUNIT,error,$log_directory/opt_$vm.xml
+                    log+=("--logger=JUNIT,error,$log_directory/opt_$vm.xml")
```

Comment:
`=` should work here as well, right? 

Just asking, doesn't need any changes.

Reply:
Sorry, for not getting back to this one earlier :)

Yeah, it would work and it would match the original code better but I think that `+=` is more appropriate here. If `log` were initialized with a non-empty array, we'd likely want to add to it rather than ovewrite it.

Reply:
No worries!